odoo/addons/base/models/ir_actions_report.py

```diff
@@ -813,3 +810,15 @@ def report_action(self, docids, data=None, config=True):
             'report_file': self.report_file,
             'name': self.name,
         }
+
+        discard_logo_check = self.env.context.get('discard_logo_check')
+        if self.env.is_admin() and not self.env.company.external_report_layout_id and config and not discard_logo_check:
```

Comment:
Looking at `web/data/res_company.xml`, a company will always have a default `external_report_layout_id`, so the modal will never be displayed.

Reply:
Yep, that's right
@odoongr Waiting instructions